website/src/components/MiniBlog/index.tsx

```diff
@@ -128,7 +129,8 @@ const MiniBlog: React.FC = () => {
     }
   }, [jsonMdData]);
 
-  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => {
+  const sortedTags = getTagsSorted(tagsDistribution,"length");
```

Comment:
This sorting also needs to be by name 
![Uploading Screenshot 2021-07-15 at 5.01.14 PM.png…]()